Source/WebCore/css/parser/CSSParserContext.cpp

```diff
@@ -89,6 +89,7 @@ CSSParserContext::CSSParserContext(const Document& document, const URL& sheetBas
     , gradientInterpolationColorSpacesEnabled { document.settings().cssGradientInterpolationColorSpacesEnabled() }
     , subgridEnabled { document.settings().subgridEnabled() }
     , masonryEnabled { document.settings().masonryEnabled() }
+    , marginTrimEnabled { document.settings().cssMarginTrimEnabled() }
```

Comment:
This shouldn't be needed since you're using `"settings-flag": "cssMarginTrimEnabled"` in `CSSProperties.json`

Reply:
Removed